Summers_James/app_base.htm

```diff
@@ -0,0 +1,150 @@
+                        <li class = " ${ 'active1' if request.dmp.page == 'list' else ''} " tabindex=7><a class="paddingright" href="/catalog/list">Products</a></li>
+                        <li class = " ${ 'active1' if request.dmp.page == 'manager' else ''} " tabindex=9><a class="paddingright" href="/manager/manager">Manager</a></li>
+                    </ul>
+        
+                        <ul class="nav navbar-nav navbar-right">
+                            % if request.user.is_authenticated:
+                            <li class="accountDisplay">hello ${request.user}</li>
+                            <li class="dropdown">                          
+                                    <a href="#" class="nav navbar-nav navbar-right dropdown-toggle" type="button" data-toggle="dropdown" tabindex=10>Account
+                                      <span class="caret"></span></a>
+                                      <ul class="dropdown-menu">
+                                        <li class = " ${ 'active1' if request.dmp.page == 'logout' else ''} "><a href="/account/logout">Logout<i class="icon-user icon-white"></i></a></li> 
+                                        <li><a href="#">My Cart</a></li>
+                                        <li class="divider"></li>
+                                        <li><a href="#">My Account</a></li>
+                                      </ul>
+                                  </li>
+                            % else :
+                                <li class = " ${ 'active1' if request.dmp.page == 'signup' else ''} "><a href="/account/signup">Sign-Up<i class="icon-user icon-white"></i></a></li>                               
+                                <li class = " ${ 'active1' if request.dmp.page == 'login' else ''} "><a href="/account/login">Login<i class="icon-user icon-white"></i></a></li> 
+                            % endif 
+                        </ul>
+                    </div>
+                  </nav>
+        </%block>
+            
+    <%block name="catalogList">
+    <div class="myListTable" class="relative">
+        <div class="displayCategoryOuter">
+            <ul class="displayCategoryInner">
+                    <li><a href="/catalog/index/">All products</a></li>
+                    % for cat in categories:
+                    <li class = " ${ 'activeUnderline' if cat.id == underline else ''} "><a href="/catalog/index/${cat.id}">${cat.name}</a></li>
+                    % endfor
+            </ul>
+        </div>    
+    </%block>
+
+    <%block name="content">
+    The list of categories will populate in here
+    </%block>
+
+    <%block name="viewed">
+
+    <div style="position: absolute; top: 200px; left: 1250px; ">
+        Recently Viewed:
+        <ol style="display:flex; flex-direction: column;">
+        %for count, i in enumerate(request.session['last_five']):
+        %if (count != 5):
```

Comment:
does this have proper indenting?

Reply:
No it doesn't. I will fix that. However, on the html side of things, the HTML peeps just don't care :)